pyk/src/pyk/kast.py

```diff
@@ -311,13 +311,53 @@ def match(self, term: KInner) -> Optional[Subst]:
 FALSE = KToken('false', BOOL)
 
 
+@final
+@dataclass(frozen=True)
+class KLabel(KInner):
+    name: str
+    params: Tuple[KSort, ...]
+
+    def __init__(self, name: str, params: Iterable[KSort] = ()):
+        object.__setattr__(self, 'name', name)
+        object.__setattr__(self, 'params', tuple(params))
```

Comment:
Can we make this take `params: Iterable[Union[KSort, str]]` instead, so that we can pass in sort parameters as strings?

Reply:
Good idea.